### ISSUE TITLE
Add parking lot counts to visualizations

### ISSUE THREAD

**Issue by sgwinder** [OutdoorRD/trails-viz#81]
This is not entirely straightforward since the parking lot counts are on a daily scale, not weekly or monthly. But would be nice to display at some point.

**Comment by EmiliaH** [comment]
Since we incorporate the parking lot count data by using it to estimate trail visitation, should we close this issue?

**Comment by sgwinder** [comment]
I still think it's potentially interesting for partners to see the parking lot counts themselves, but I don't see it as a priority. So yes, I'll close it here.

**Comment by EmiliaH** [comment]
Cool, thanks @sgwinder! The other day when David and I were talking about data viz, I had the thought that displaying parking lot counts over the course of a day could be interesting, or something along these lines. In other words use these parking lot count data to display parking capacity over a day scale. 

This feels like it's outside the scope of this project but could be potentially fun and interesting. And *maybe* useful for management. Anyway, if you agree, we could make an issue for this idea or reopen this idea.

**Comment by sgwinder** [comment]
Yes - that's similar to my initial thought with this issue - that it might be relevant to see something about the max number of cars recorded at various trails. And you're right that this would be more meaningful w/ times of day included... the parking lot counts are actually at the "instantaneous" scale rather than the "daily" scale.

I'll reopen, but definitely agree that this is just a bonus idea until we have a stronger request for it from somewhere.
